src/component/registry/bpmn-elements-registry.ts

```diff
@@ -29,7 +29,7 @@ export function newBpmnElementsRegistry(bpmnModelRegistry: BpmnModelRegistry, gr
 }
 
 /**
- * @category Interaction
+ * @category Custom Behavior
```

Comment:
I am not sure this is the best name, but at least it is align we the concepts we have documented.

Reply:
I think it is totally OK